tutor/src/components/onboarding/ux/student-course.ts

```diff
@@ -75,7 +104,7 @@ export class StudentCourseOnboarding extends BaseOnboarding {
     @action.bound
     onPaymentComplete() {
         if (this.paymentIsPastDue) {
-        // in this case we have to reload since network requests have been failing silently
+            // in this case we have to reload since network requests have been failing silently
             setTimeout(forceReload);
         } else {
             this.displayPayment = false;
```

Comment:
hm, I'm curious if this is still needed?

Reply:
Tested removing it locally and it seems to no longer be needed? I'll commit it and we can test more on staging.